### ISSUE TITLE
How to start this character sheet locally

### ISSUE THREAD

**Issue by bragma** [mraichelson/cyberpunk-red-tabbed#26]
Hi,
I'd like to help out development or at least try to fix some of the minor bugs. I've forked the repo, installed node packages and started webpack from npm. If I serve the dist folder from my webserver, the page is broken: no style is applied and tabs do not work. any suggestion on what I may be doing wrong?
Thanks!

**Comment by bragma** [comment]
Sorry for the stupid question, I've found my answer just googling around. Are you open to PR?

**Comment by mraichelson** [comment]
Yeah, the webpack local server code doesn't actually serve the character sheet for preview, I should definitely add some notes around that in the Readme. Glad to take PRs.

**Comment by bragma** [comment]
Thanks for this sheet, I'm a real beginner with a lot to learn and I'd like to start from a good base. I'll send you a frw trial PRs for the easiest fixes I've found. In any case my main goal here is to try to compat the sheet to make it fit my screen. I have an high res monitor, still there are so many skills that I need to scroll too much for my taste. So I've played with css to remove some borders and margins and reordered skill boxes to decrease blank space.
Let me know if you are interested in it, I understand it's a matter of taste so no offense taken.

![image](https://user-images.githubusercontent.com/1101058/102938181-16a66780-44ac-11eb-877e-86a13477611c.png)

**Comment by mraichelson** [comment]
Since I was in there and poking at things I did replicate some of the spacing adjustments you were talking about. I did a bit of adjustment around button sizing and stuff to help make things wrap a bit (especially if you're a Nomad with Moot being added to skills as well the text in the buttons starts to get very long) and help keep things aligned a bit. But I do see what you mean about the groups needing to get shuffled around. (The dotted pink outlines are temporary for showing where the boundaries of the buttons lie.)

<img width="910" alt="Screen Shot 2021-01-02 at 1 38 17 PM" src="https://user-images.githubusercontent.com/70476/103465309-dcd12e80-4cff-11eb-98b4-de46010a0d7c.png">

**Comment by mraichelson** [comment]
I know it's not much of a change to re-order those groups, but there's still this mild twinge of "if they're not alphabetical does it make it even harder to find a particular skill?" there in the back of my head.

**Comment by mraichelson** [comment]
My current state of experimenting with this is in the [`feature/skills-3col`](https://github.com/mraichelson/cyberpunk-red-tabbed/tree/feature/skills-3col) branch.

**Comment by bragma** [comment]
Hi, sorry for the late reply. I've had some time to test with the 3 col layout and I think it's a big win. I hope you'll merge it in the main branch, so I could ditch my fork.

**Comment by vokaplok** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!
